docs/design.rst

```diff
@@ -8,51 +8,48 @@ Overview
-            =          |                |          =
-            =       ========================       =
-            =       = Babble Proxy         =       =
-            =       ========================       =
-            =          |                ^          =
-            ===========|================|===========
-                       |                |
-            --------- SubmitTx(tx) ---- CommitBlock(Block) ------- (JSON-RPC/TCP)
-                       |                |
-         ==============|================|===============================
-         = BABBLE      |                |                              =
-         =             v                |                              =
-         =          ========================                           =
-         =          = App Proxy            =                           =
-         =          =                      =                           =
-         =          ========================                           = 
-         =                     |                                       =
-         =   =======================================                   =
-         =   = Core                                =                   =
-         =   =                                     =                   =
-         =   =                                     =    ============   =
-         =   =  =============        ===========   =    = Service  =   =  
-         =   =  = Hashgraph =        = Store   =   = -- =          = <----> HTTP API
-         =   =  =============        ===========   =    =          =   =
-         =   =                                     =    ============   =     
-         =   =                                     =                   =
-         =   =======================================                   =
-         =                     |                                       =
-         =   =======================================                   =
-         =   = Transport                           =                   =
-         =   =                                     =                   =
-         =   =======================================                   =
-         =                     ^                                       =
-         ======================|========================================
-                               |                  
-                               v
-                       
-                            Network
+        +--------------------------------------+
+        | APP                                  |
+        |                                      |
+        |  +-------------+     +------------+  | 
+        |  | Service     | <-- | State      |  |
+        |  |             |     |            |  |
+        |  +-------------+     +------------+  |
+        |          |                ^          |
+        |          |                |          |
```

Comment:
I would have let the BabbleProxy here to explain graphically the two-part implementation of the proxy